test/HealthChecks.AzureServiceBus.Tests/AzureServiceBusTopicHealthCheckTests.cs

```diff
@@ -0,0 +1,253 @@
+using Azure.Core;
+using Azure.Messaging.ServiceBus.Administration;
+using HealthChecks.AzureServiceBus.Configuration;
+using NSubstitute;
+using NSubstitute.ExceptionExtensions;
+
+namespace HealthChecks.AzureServiceBus.Tests;
+
+public class azureservicebustopichealthcheck_should
+{
+    private readonly string ConnectionString;
+    private readonly string FullyQualifiedName;
+    private readonly string TopicName;
+    private readonly string HealthCheckName = "unit-test-check";
```

Comment:
```suggestion
    private const string HEALTH_CHECK_NAME = "unit-test-check";
```

here and in other files